airbyte-cdk/python/airbyte_cdk/sources/declarative/schema/static_schema_loader.py

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import InitVar, dataclass
+from typing import Any, Dict, Mapping
+
+from airbyte_cdk.sources.declarative.schema.schema_loader import SchemaLoader
+
+
+@dataclass
+class StaticSchemaLoader(SchemaLoader):
```

Comment:
Something I'm thinking about is whether `StaticSchemaLoader` is the right name for this component. Static feels like too vague a term because technically all our connector's that reference some underlying `example_schema.json` are also static, just not living within the manifest file itself.

`InlineSchemaLoader(SchemaLoader)` might be a good alternative since what we want to highlight from the component is that its defined within the manifest, not that its static.

Reply:
Great suggestions, @brianjlai - I like `InlineSchemaLoader` and will update the name in the relevant places.

I'll also update the example inline schemas in https://github.com/airbytehq/airbyte/pull/20405 to use a top-level `schemas` key. I believe that will require a small change to this PR.

Reply:
Pushed the suggested changes to this PR and https://github.com/airbytehq/airbyte/pull/20405.